code/server/src/models/database.ts

```diff
@@ -1,18 +1,22 @@
 import { FieldInfo, MysqlError, queryCallback } from 'mysql';
 
-import { connection } from './connection';
+import { Connection } from './connection';
 import { Logger } from '../helpers/logger';
 import { DB_LOG } from '../constants/logs';
 
 export const database = {
   query: (sql: string, cb: queryCallback) => {
-    connection.query(sql, (err: MysqlError | null, results?: any, fields?: FieldInfo[]) => {
+    Connection.get().query(sql, (err: MysqlError | null, results?: any, fields?: FieldInfo[]) => {
       Logger(DB_LOG, `${sql}; (${(err && err.message) || 'OK'})\n`);
+      if (err && err.fatal) {
```

Comment:
Use the error handler from connection.ts.

Reply:
Fixed [here](https://github.com/Fezzzi/Household-Chores/pull/10/commits/df91c2a3b76cb0477266a4db46e1397cbb108b9f#diff-4acfd404455a1add895ffa6ef4bd1dec)